subscription_platform/dashboards/monitor_saasboard__revenue.dashboard.lookml

```diff
@@ -0,0 +1,661 @@
+      display: popover
+    model: subscription_platform
+    explore: monthly_active_logical_subscriptions
+    listens_to_filters: [Service ID]
+    field: monthly_active_logical_subscriptions.subscription__payment_provider
+  - name: Plan Interval
+    title: Plan Interval
+    type: field_filter
+    default_value: ''
+    allow_multiple_values: true
+    required: false
+    ui_config:
+      type: checkboxes
+      display: popover
+    model: subscription_platform
+    explore: monthly_active_logical_subscriptions
+    listens_to_filters: [Service ID]
+    field: monthly_active_logical_subscriptions.subscription__plan_interval
+  - name: Plan
+    title: Plan
+    type: field_filter
+    default_value: ''
+    allow_multiple_values: true
+    required: false
+    ui_config:
+      type: checkboxes
+      display: popover
+    model: subscription_platform
+    explore: monthly_active_logical_subscriptions
+    listens_to_filters: [Service ID]
+    field: monthly_active_logical_subscriptions.subscription__plan_summary
+  - name: Region
+    title: Region
+    type: field_filter
+    default_value: ''
+    allow_multiple_values: true
+    required: false
+    ui_config:
+      type: checkboxes
+      display: popover
+    model: subscription_platform
+    explore: monthly_active_logical_subscriptions
+    listens_to_filters: [Country, Service ID]
+    field: countries.region_name
+  - name: Country
+    title: Country
+    type: field_filter
+    default_value: ''
+    allow_multiple_values: true
+    required: false
+    ui_config:
+      type: checkboxes
+      display: popover
+    model: subscription_platform
+    explore: monthly_active_logical_subscriptions
+    listens_to_filters: [Service ID, Region]
+    field: countries.name
+  - name: Has Refunds (Yes / No)
+    title: Has Refunds (Yes / No)
+    type: field_filter
+    default_value: ''
+    allow_multiple_values: true
+    required: false
+    ui_config:
+      type: dropdown_menu
+      display: overflow
+    model: subscription_platform
+    explore: monthly_active_logical_subscriptions
+    listens_to_filters: []
+    field: current_subscription_state.has_refunds
+  - name: Has Fraudulent Charges (Yes / No)
+    title: Has Fraudulent Charges (Yes / No)
+    type: field_filter
+    default_value: 'No'
+    allow_multiple_values: true
+    required: false
+    ui_config:
+      type: dropdown_menu
+      display: overflow
+    model: subscription_platform
+    explore: monthly_active_logical_subscriptions
+    listens_to_filters: []
+    field: current_subscription_state.has_fraudulent_charges
+  - name: Service ID
+    title: Service ID
+    type: field_filter
+    default_value: Monitor
+    allow_multiple_values: true
+    required: false
+    ui_config:
+      type: advanced
+      display: overflow
+      options: []
+    model: subscription_platform
+    explore: monthly_active_logical_subscriptions
+    listens_to_filters: []
+    field: subscription_services.id
```

Comment:
The service ID filter should be required, and "Monitor" should be the only allowed value.
```suggestion
  - name: Service ID
    title: Service ID
    type: field_filter
    default_value: Monitor
    allow_multiple_values: true
    required: true
    ui_config:
      type: advanced
      display: overflow
      options:
      - Monitor
    model: subscription_platform
    explore: monthly_active_logical_subscriptions
    listens_to_filters: []
    field: subscription_services.id
```